login/resources/css/login.css

```diff
@@ -370,6 +371,7 @@ a.zocial  {
     padding: 20px 60px 0 0;
     color: #72767b;
     letter-spacing: 0;
+    margin-top: 50px;
```

Comment:
Can this be dynamic and based on a viewport height? So if the page is zoomed in - everything is still visible without scrolling down